change/@itwin-grouping-mapping-widget-e8e9e183-2074-4294-8bbe-84cd626f54c6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "removed export for calculated and custom property",
```

Comment:
Can you be more specific about what changed? What replaced these two? This is also a `minor` change. We have not started using `major` bumps.

Reply:
Done.